rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- [ ] The rentals suppose to add itself to @book and @person in order to achieve the `belongs-to` association according to the project requirement. ℹ️ Kindly implement it. 👍 For example:
```
    book.rentals << self    
```